src/web/app/pages-help/instructor-help-page/instructor-help-page.component.ts

```diff
@@ -22,14 +24,22 @@ export class InstructorHelpPageComponent implements OnInit, AfterViewInit {
 
   @ViewChild('helpPage') bodyRef ?: ElementRef;
 
-  constructor(private route: ActivatedRoute) { }
+  constructor(private route: ActivatedRoute,
+              private pageScrollService: PageScrollService,
+              @Inject(DOCUMENT) private document: Document) { }
 
   ngOnInit(): void {
   }
 
   ngAfterViewInit(): void {
-    this.route.fragment.subscribe((f: string) => {
-      this.scroll(f);
+    this.route.queryParams.subscribe((queryParam: Params) => {
+      if (queryParam.questionId) {
+        setTimeout(() => this.pageScrollService.scroll({
+          document: this.document,
+          scrollTarget: `#${queryParam.questionId}`,
+          scrollOffset: 70,
+        }), 500);
```

Comment:
- The scrolling actions seems a bit too slow (based on the screenshot provided)
- Just scrolling is not sufficient; the relevant help section needs to be expanded